Mobile/iOS/Templates/Release_3.4.7/xxx-Federate_sharing.md

```diff
@@ -0,0 +1,55 @@
+| 16 |  Deleted account |  1. Repeat the test cases 2. or 3.<br> 2. From admin in web, delete the account ot the sharing user | The sharee can not view the shared file or folder |||
+| 17 |  Without connection |  1. Switch the wifi connection off<br>2. Try to share with a federated user | An error appears |||
+| 18 |  Without server connection |  1. Switch the server off<br>2. Try to share with a federated user | An error appears |||
+| **Server Capabilities** ||||||
+| 19 |  Federated Sharing disabled |  1. In server side, disable the capability to share with federation<br>2. In app, try to share with federation<br> | The share is not accepted and an error is raised |||
+| 20 |  Receive Federated disabled |  1. In server side, disable the capability to share with federation<br>2. In app, try to share form a other server's user with the current server<br> | The share is not accepted and an error is raised |||
+| **Edit Privilege** ||||||
+| 21 | Edit granted file|  1. Swipe to share a file with users<br>2. Type a federated URL<br>3. Grant the federated user to edit<br> 4. Login in web with the sharee and acept the federated share<br>5. Login in app with the sharee and try to edit the file  | The shared file appears in files view and can be edited |||
+| 22 | Edit granted folder|  1. Swipe to share a folder with users<br>2. Type a federated URL<br>3. Grant the federated user to edit<br> 4. Login in web with the sharee and acept the federated share<br>5. Login in app with the sharee and try to edit the folder  | The shared folder appears in files view and can be edited (upload/update/delete) |||
+| 23 | Edit not granted file |  1. Swipe to share a file with users<br>2. Type a federated URL<br>3. Do not grant the federated user to edit<br> 4. Login in web with the sharee and acept the federated share<br>5. Login in app with the sharee and try to edit the file  | The shared file appears in files view and can not be edited |||
+| 24 | Edit not granted folder |  1. Swipe to share a folder with users<br>2. Type a federated URL<br>3. Do not grant the federated user to edit<br> 4. Login in web with the sharee and acept the federated share<br>5. Login in app with the sharee and try to edit the folder  | The shared file appears in files view and can not be edited (upload/update/delete)|||
+| **Reshare Privilege** ||||||
+| 25 | Reshare in the server |  1. Swipe to share a file with an federated user granting him to share (defect)<br>2. Login with the sharee<br>3. Try to reshare with other users in the same server<br>4. Login with the 2nd sharee| 3. The share privilege is enabled<br>4. The 2nd sharee can view the file |||
+| 26 | Reshare federated |  1. Swipe to share a file with an federated user granting him to share<br>2. Login with the sharee<br>3. Try to reshare with other federated user<br>4. Login with the 2nd sharee| 3. The share privilege is enabled and can not be disabled<br>4. The 2nd sharee can view the file |||
```

Comment:
ups, copy-paste the previous test case 
suggestion: 
| 21 |  Receive Federated disabled |  1. In server side, disable the capability to receive federated data on server1 <br>2. In desktop, try to share with a user from server2 with a user from server1<br> | The share is not accepted and an error is raised |||